aries-site/src/pages/components/tag.mdx

```diff
@@ -0,0 +1,77 @@
+import { Example } from '../../layouts';
+import { TagAnatomy, TagBasicAnatomy } from '../../examples';
+import { FormClose } from 'grommet-icons';
+
+## Guidance
+
+Tags are simple attributes containing metadata:
+
+- Simple, brief and concise metadata to describe a resource or content
+- Included in search, filter, and in the details of a resource
+- Organize and group resources by common characteristics or attibutes
+- Assisst batch actions and selection of multiple resources
+
+## Anatomy
+
+The most basic tags consist of two primary elements:
+
+<Example plain>
+  <TagBasicAnatomy />
+</Example>
+
+1. Border
+1. Name
+
+Tags can have additional elements depending on the tags’ intended use and context:
+
+<Example plain>
+  <TagAnatomy />
+</Example>
+
+1. Border
+1. Name
+1. Value
+1. Close icon
+
+Character requirements:
+
+- Name : 1-128 characters
+- Value : 1-256 characters
```

Comment:
Space preceding colon should not be here.
```suggestion
- Name: 1-128 characters
- Value: 1-256 characters
```

Also, for some reason the '1-128' is causing me to have to think more - I'm feeling a heavier cognitive load because I recognize there is a range, I skim it, glossing over the start value, anchor on the '256 characters', and then my eye has to backtrack to remind myself what the start value is and its simply '1'. May be just me, but does this flow easier?:
```suggestion
- Name: up to 128 characters
- Value: up to 256 characters
```

Reply:
I like "up to", implementing.